app/Main.hs

```diff
@@ -1,13 +1,25 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
LANGUAGE pragmas changes the compiler steps. OverloadedStrings is making it so we can write `"hello world"` and that can be a `String` or `Text` depending on how it is used.

The RecordWildCards is letting use use the line `return AppSettings{..}` down below which translates to `return AppSettings { appPort = appPort }`. As we add more settings to `AppSettings` using the wildcards will allow us to leave that last line alone.